### ISSUE TITLE
Is there a plan to switch from wxT("") to wxString::FromUTF8?

### ISSUE THREAD

**Issue by alex290** [wxFormBuilder/wxFormBuilder#796]
Hello, wxT("...") is everywhere in the code generator. In version 3.2, they removed it from the documentation on the forum, they wrote that they left it only for backward compatibility and recommend using wxString::FromUTF8 (https://docs.wxwidgets.org/3.2.4/overview_string.html). Is there a replacement planned in wxFormBuilder?

**Comment by Kumazuma** [comment]
I think that wxT should be replaced to wxS, not wxString::FromUTF8.

**Comment by alex290** [comment]
> I think that wxT should be replaced to wxS, not wxString::FromUTF8. https://docs.wxwidgets.org/3.2.4/group__group__funcmacro__string.html#ga0a06701ea0e6bb00c5171ace1b1f5691

I agree, it is possible. Since the documentation recommends not using wxT

https://docs.wxwidgets.org/3.2.4/group__group__funcmacro__string.html#ga437ea6ba615b75dac8603e96ec864160

![4FlpJau 1](https://github.com/wxFormBuilder/wxFormBuilder/assets/13647764/2aa3fff0-dac9-4430-abfe-4813f8965c68)

**Comment by sodevel** [comment]
What exactly do you mean by "is everywhere in the code generator"? The wxFB source code itself or the generated code?

When i am rewriting wxFB source code, i am removing `wxT()` when i encounter them or replace them with `_()`, because in many parts of the code these two marcos are used wrong and their usage needs to be switched.

In generated code you should rarely see this macro, unless you have disabled internationalization which you usually should not do. Although the wxWidgets documentation says you should not use this macro anymore, it also mentions a huge limitation if you do so: you should only use 7 bit characters. While this is not a big issue for the english language, it is for other languages.

I encountered many encoding related problems in the past, especially Windows is prone for such issues. Even in these days, the safest way to avoid them is using wide strings. Since `wxT("")` is nothing more than `L""` today, i currently have no plans to switch from `wxT()` to anything else.

**Comment by alex290** [comment]
I only meant code generation. As we wrote above, wxS("") is what wxWidgets recommend in the documentation.
